builtin/game/register.lua

```diff
@@ -155,6 +155,8 @@ end
 local function preprocess_craft(itemdef)
 	-- BEGIN Legacy stuff
 	if itemdef.inventory_image == nil and itemdef.image ~= nil then
+		core.log("deprecated", "The `image` field in craftitem definitions " ..
+			"is deprecated. Use `inventory_image` instead.")
```

Comment:
will this correctly identify the caller or how would modders identify the affected code?

Reply:
Yes. It looks like this:

> 2025-04-01 16:27:15: WARNING[ServerStart]: The `image` field in craftitem definitions is deprecated. Use `inventory_image` instead. (at /home/lars/luanti/mods/dev/init.lua:1)

If you have "info" log level or higher, you will see a full stack trace, but this is typically not necessary.